src/js/controllers/imageCtrl-test.js

```diff
@@ -0,0 +1,125 @@
+describe('imagesCtrl', function() {
+    var $rootScope, $controller, imagesCtrl, images, vm, $q;
+
+    beforeEach(module('flickrPOC'));
+
+    beforeEach(module(function($provide){
+        $provide.service('images', function($q){
+            this.loadPics = sinon.stub().returns($q.resolve({message: 'loadPics Working fine'}));
+        });
+
+    }));
+
+    beforeEach(inject(function (_$rootScope_, _$controller_, _images_, _$q_) {
+        $rootScope = _$rootScope_.$new();
+        $controller = _$controller_;
+        images = _images_;
+        $q = _$q_;
+
+        vm = $controller('imagesCtrl');
+
+    }));
+
+    describe('imagesCtrl', function(){
+        beforeEach(function () {
+            images.loadPics.returns($q.resolve(
+                {
```

Comment:
Can we move this return object to a globally available variable? That way any test can reference, if needed. You'll need to declare it with your other top-level variables, before the first `beforeEach()`.